src/modules/file/libkvifile.cpp

```diff
@@ -1216,19 +1216,30 @@ static bool file_kvs_cmd_writeLines(KviKvsModuleCommandCall * c)
 	@syntax:
 		<string> $file.localdir([relative_path:string])
 	@description:
-		Returns the path to the KVIrc local data directory.[br][br]
+		Returns the path to the KVIrc local data directory.
 		The KVIrc local data directory is always writable and contains
 		the various subdirectories that KVIrc uses internally: audio, avatars,
-		config, help, incoming, log, modules, msgcolors and pics.[br][br]
-		If <relative_path> is passed, then it is appended to the local directory path.[br][br]
+		config, help, incoming, log, modules, msgcolors and pics.
+		[br][br]
+		If <relative_path> is passed, then it is appended to the local directory path.
 		The path is adjusted to contain single separators suitable for the platform
 		that KVIrc is actually running on (thus you do not need to care about path
-		separators in the <relative_path>, KVIrc will adjust them).[br]
+		separators in the <relative_path>, KVIrc will adjust them).
+		[br][br]
+		When adding popup entries especially via script using [b]popup.addExtPopup[/b]
+		you should escape the path [b]$escape($file.localdir(pics/panic.png))[/b]
+		this is multiOS friendly and adds a custom icon path to Windows that is not broken
+		when KVS parses the context menu entry due to Windows paths being backslashed,
+		which are special characters in KVS that require escaping.[br]
+		Overall [b]$escape($file.localdir(pics/panic.png)) is a more correct approach.
 	@examples:
 		[example]
 			echo KVIrc looks for pictures in $file.localdir(pics)
 			echo panic.png would be translated to $file.localdir(pics/panic.png)
+			echo panic.png would be translated to $escape($file.localdir(pics/panic.png))
```

Comment:
Why both lines, without `$escape` and with it?

Reply:
I dont know, maybe because you dont need to use with in every case.

Reply:
You *need* to use it in every case, unless you want to check if you're on Windows or on Linux before adding a popup menu, which is something you should not do.

The best approach once again is to use `\$` because that way you can move your configuration folder between OSes and have it working, but otherwise `$escape` is a must.

Reply:
Thats not what I found at all and I and others have tested this,

what you suggest is broken in some cases, and I wont add more broken docs.
feel free to close.

Reply:
That sounds like a un1versal solution.

Reply:
> That sounds like a un1versal solution.

This is the second time youre making some implications that are false.
I know the person know here as un1versal, he was a very good friend of my father.

Not that, I owe you or anyone any explanations, but un1versal was a good person and did nothing afaik but help. He's passed away now for some time and so has my father, and unless you have something against me (which escapes me because I dont know you or you me), I have nothing else to say.

Reply:
I don't have anything against you, I just don't believe the story.

Reply:
I have nothing to prove to you or anyone and have done nothing to you or this project but try to help a friend of my father with some of his projects and this is already a waste of time.

Your attitude here of making petty reports to Github are a waste of my time and hinder contributions. If you dont want my contributions just close them,

Reply:
I didn't report anything to GitHub and let's leave it at, we would not have a problem if un1versal returned, no need for games.

Reply:
Tread carefully, he was a friend of my fathers, perhaps his best and oldest friend, dont disrespect the dead by making false implications.
I have great respect for the man he was and what he stood for, hes passed away long ago so let that rest already.